pyignite/dbapi/sqlalchemy.py

```diff
@@ -0,0 +1,281 @@
+#
```

Comment:
I am strongly against to put dependency on sqlalchemy to thin client.

Reply:
I don't think it should be a _mandatory_ requirement, but support for SQLAlchemy is important. I may yet break this out into a separate PR.

Reply:
I suppose that it should be additional module. Just start separate module in separate repo.

Reply:
We can of course rearrange repo, but it is not a hood idea, imho. Currently there is not any options to make it "optional". Just create new module pyignite-sqlalchemy and develope it independently.

Reply:
And sqlalchemy is not important at all, it is just one of many others ORM, and you can barely gind any db driver, that contains anything related to ORM in main repo